docs/deprecations.md

```diff
@@ -386,3 +386,11 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 Deprecation code: `BIN_SERVERLESS`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="DEFAULT_NODEJS12X_RUNTIME_DEPRECATED"><div>&nbsp;</div></a>
+
+## Default runtime nodejs12.x deprecated
+
+Deprecation code: `DEFAULT_NODEJS12X_RUNTIME_DEPRECATED`
+
+Please use `nodejs14.x` as the runtime as it will be configure as the default runtime in the next major release (3.0.0).
```

Comment:
This message is a bit misleading - we want to inform people that if they don't set the runtime explicitly, starting with next major it will change to `nodejs14.x`, so if they want to be certain that they use consistent runtime, they should set it explicitly, not necessarily to `nodejs14.x`, they can still set it to `nodejs12.x` for example. 

Given that, what do you think about such messaging:

```
Starting with `v3.0.0`, the default runtime will change from `nodejs12.x` to `nodejs14.x`. In order to hide the deprecation message and ensure seamless upgrade, please set the runtime explicitly.
```

What do you think?